x/evm/keeper/statedb_test.go

```diff
@@ -476,9 +475,60 @@ func (suite *KeeperTestSuite) TestEmpty() {
 }
 
 func (suite *KeeperTestSuite) TestSnapshot() {
-	revision := suite.app.EvmKeeper.Snapshot()
-	suite.Require().Zero(revision)
-	suite.app.EvmKeeper.RevertToSnapshot(revision) // no-op
+
+	var key = common.BytesToHash([]byte("key"))
+	var value1 = common.BytesToHash([]byte("value1"))
+	var value2 = common.BytesToHash([]byte("value2"))
+
+	// set to the initial context
+	suite.Run("simple revert", func() {
```

Comment:
please use [table driven tests](https://ieftimov.com/post/testing-in-go-table-driven-tests/). You can check the examples on the other test files

Reply:
done